fundmanager/fundmanager.go

```diff
@@ -56,51 +56,71 @@ func New(cfg Config) func(api v1api.FullNode, sqldb *sql.DB) *FundManager {
 	}
 }
 
+type TagFundsResp struct {
+	ForCollat  abi.TokenAmount
+	ForPublish abi.TokenAmount
+
+	TotalTaggedCollat  abi.TokenAmount
+	TotalTaggedPublish abi.TokenAmount
+
+	RemainingCollat  abi.TokenAmount
+	RemainingPublish abi.TokenAmount
```

Comment:
nit: I suggest renaming these for consistency with how they're used in the code base:
ForCollat -> Collateral
ForPublish -> PublishMessage
TotalTaggedCollat -> TotalCollateral
TotalTaggedPublish -> TotalPublishMessage
RemainingCollat -> AvailableCollateral
RemainingPublish -> AvailablePublishMessage